README.md

```diff
@@ -128,19 +128,39 @@ Following these step, both servers MAY display the `name` of the other party as
 
 For further details on this concept, see also [#54](https://github.com/cs3org/OCM-API/pull/54) and related issues. For a discussion about trust policies, see [sciencemesh#196](https://github.com/sciencemesh/sciencemesh/issues/196).
 
-### Discovery
-Authentication between services is already established. This means that this specification doesn't cover the way a service authenticates incoming API calls (e.g. through an API Key, VPN connection or IP whitelisting). In this scope we assume that the services are already authenticated.
+### OCM API Discovery
+After establishing contact as discussed in the previous section, the Sharing User can send the Share Creation Gesture to the Sending Server, providing the Sending Server with the following information:
+* Resource to be shared
+* Protocol to be offered for access
+* Sending Party's identifier
+* Receiving Party's identifier
+* Receiving Server FQDN
+* OPTIONAL: Name
```

Comment:
Based on https://github.com/cs3org/OCM-API/pull/108 isn't name a required field?

Reply:
moving it to the other PR https://github.com/cs3org/OCM-API/pull/122